src/main/kotlin/app/revanced/patches/youtube/layout/theme/patch/ThemePatch.kt

```diff
@@ -24,28 +23,164 @@ class ThemePatch : ResourcePatch {
         val darkThemeBackgroundColor = darkThemeBackgroundColor!!
         val lightThemeBackgroundColor = lightThemeBackgroundColor!!
 
-        context.xmlEditor["res/values/colors.xml"].use { editor ->
-            val resourcesNode = editor.file.getElementsByTagName("resources").item(0) as Element
+        val android11ValuesFolder = "values"
+        val android12ValuesFolder = "values-v31"
+        val drawableFolders: List<String> = listOf(
+            "drawable",
+            "drawable-sw600dp"
+        )
+        val splashScreenColorAttributeName = "splashScreenColor"
```

Comment:
Why is this a field? The field name already explains what it is.

Reply:
There are some methods that use this name as parameters (3 in total). I decided to convert it to a field to deduplicate the code.

Reply:
If there are multiple methods using this same attribute, then the code should be deduplicated. A class field does not make sense for this.